src/winetricks

```diff
@@ -7865,6 +7866,32 @@ load_icodecs()
 
 #----------------------------------------------------------------
 
+w_metadata cinepak dlls \
+    title="Cinepak Codec" \
+    publisher="Radius" \
+    year="1995" \
+    media="download" \
+    file1="cvid32.zip" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/iccvid.dll" \
+    homepage="http://www.probo.com/cinepak.php"
+
+load_cinepak()
+{
+    w_download "http://www.probo.com/pub/cinepak/${file1}" a41984a954fe77557f228fa8a95cdc05db22bf9ff5429fe4307fd6fc51e11969
```

Comment:
Please don't use $file1 inside of w_download, it breaks linkcheck.sh

Reply:
done